### PR TITLE
Redesign schedule feature in calendar section 

### PR DESCRIPTION
**Pull request by coder12git**
Redesign the schedule feature of Mentor Dashboard.

### CODE REVIEW

**Comment by coder12git** [comment]
Screenshot 2023-08-05 165802
@anmode please review this

**Comment by anmode** [comment]
Cool but need some fixes:
1. Mobile optimization
2.  
Screenshot 2023-08-05 at 5 05 40 PM

btw yeh calender dikh ni rha? Updated ni kri pR

File: components/mentorDashboard/calendar.js [hunk 1]
```diff
@@ -1,22 +1,23 @@
 import React, { useState } from "react";
 import styles from "../../styles/dashboard.module.css";
```
Comment (anmode) [inline components/mentorDashboard/calendar.js:2]: actually yeh bdia h ... schedule m 1. schedule 2. Calender
and routing kch aisi kr lenge `tab=schedule&subTab=scedule` or `tab=schedule&subTab=calender ` how it is?

**Review by anmode — APPROVED** [review]

**Comment by coder12git** [comment]
> Cool but need some fixes:
> 
> 1. Mobile optimization
> 
> Screenshot 2023-08-05 at 5 05 40 PM
> btw yeh calender dikh ni rha? Updated ni kri pR

Ok will do this

File: components/mentorDashboard/calendar.js (hunk 1 above)
Comment (coder12git) [reply components/mentorDashboard/calendar.js:2]: > actually yeh bdia h ... schedule m 1. schedule 2. Calender and routing kch aisi kr lenge `tab=schedule&subTab=scedule` or `tab=schedule&subTab=calender ` how it is?

Isn't is same?

**Comment by coder12git** [comment]
@anmode fixed the time and alignment issue, but I will not able to make it mobile-friendly.

**Comment by coder12git** [comment]
Screenshot 2023-08-05 220449

**Comment by anmode** [comment]
No worries you did great btw... keep contributing

**Pull request merged**